libs/sdk-ui-pivot/src/CorePivotTable.tsx

```diff
@@ -714,9 +710,11 @@ export class CorePivotTablePure extends React.Component<ICorePivotTableProps, IC
     // column resizing stuff
     //
 
-    private isColumnAutoresizeEnabled = (props: ICorePivotTableProps = this.props) => {
-        return this.getDefaultWidthFromProps(props) === "viewport";
-    };
+    private isColumnAutoresizeEnabled = () =>
+        this.getDefaultWidthFromProps(this.props) === "viewport" || this.isColumnAutoresizeAllEnabled();
+
+    private isColumnAutoresizeAllEnabled = () =>
+        this.getDefaultWidthFromProps(this.props) === "autoresizeAll";
```

Comment:
I thought that the impl will not distinguish between viewport / autoresizeAll.

why do we need these two methods then?

Reply:
Done. I removed isColumnAutoresizeAllEnabled method.